LowerSchool.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document</title>
+</head>
+
+<body>
+  <nav>
+    <ul>
+      <li>
+        <a href="index.html">Home</a>
+      </li>
+      <li>
+        <a href="lowerschool.html">Lower School</a>
+      </li>
+      <li>
+        <a href="upperschool.html">Upper School</a>
+  </nav>
+  <ul></ul>
+  <li>18 months</li>
+  <li>2s-3s</li>
+  <li>Junior Kindergarten</li>
+  <li>Kindergarten</li>
+  <li>1st grade</li>
+  <li>2nd grade</li>
+  <li>3rd grade</li>
+  <li>4th grade</li>
+  <li>5th grade</li>
+
+</body>
+
+</html>
+
+<h1>Lower School</h1>
+<h6>The Lower School includes the preschool and the elemntary grades, 18 months through 5th grade.</h6>
```

Comment:
You should not skip heading levels. This should be an `<h2>` instead. You can adjust the sizing of the font with CSS.